occurrence-ws/src/main/java/org/gbif/occurrence/ws/OccurrenceWsApplication.java

```diff
@@ -55,6 +54,7 @@
       "org.gbif.ws.remoteauth",
       "org.gbif.ws.security",
       "org.gbif.occurrence.search",
+      "org.gbif.occurrence.search.configuration",
```

Comment:
isn't this package already scanned by the line above?

Reply:
yes. removed